index.html

```diff
@@ -54,8 +54,211 @@ <h2>LET'S CONNECT</h2>
+            <div class="langs">
+              <p>Canopy</p>
+              <ul>
+                <li>
+                  <div class="div"></div>
+                </li>
+                <span>Full Stack Dev</span>
+                <li>
+                  <div class="div"></div>
+                </li>
+                <span>2015</span>
+              </ul>
+            </div>
+          </div>
+    
+          <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+          <div class="tech">
+            <ul>
+              <li><span>HTML</span> </li>
+              <li><span>CSS</span> </li>
+              <li><span>Javascript</span> </li>
+            </ul>
+          </div>
+        </div>
+        <button type="submit" class="btn">See Projects</button>
+      </article>
+    </section>
 
+  
+    <section class="container-b">
+      <article class="card-b">
+        <h1 class="about-h1">About Myself</h1>
+  
+        <h3 class="about-p">
+          I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and
+          experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
+        </h3>
+  
+        <h3 class="connect">LET'S CONNECT</h3>
+  
+        <ul class="socials">
+          <li>
+            <a href="https://twitter.com/ejama_isaac">
+              <img src="img/Vector (3).svg" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/ejama-isaac-bab607267/"><img src="img/Linkedin icon.svg" alt=""></a>
+          </li>
+          <li>
+            <a href="#"><img src="img/Vector (2).svg" alt=""></a>
+          </li>
+          <li>
+            <a href=""><img src="img/Vector.svg" alt=""></a>
+          </li>
+          <li>
+            <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
+          </li>
+        </ul>
+        <button class="btn-b">Get my resume</button>
+  
+        <div class="techs">
+          <h3 class="under-lines">Languages <i class="fa-solid fa-chevron-down solid"></i></h3>
+  
+          <li class="lis"><img class="img" src="https://upload.wikimedia.org/wikipedia/commons/3/3b/Javascript_Logo.png"
+              alt="">Javascript</li>
+          <li class="lis"><img class="img"
+              src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/61/HTML5_logo_and_wordmark.svg/1200px-HTML5_logo_and_wordmark.svg.png"
+              alt="">HTML</li>
+          <li class="lis"><img class="img"
+              src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d5/CSS3_logo_and_wordmark.svg/1200px-CSS3_logo_and_wordmark.svg.png"
+              alt="">CSS</li>
+  
+          <h3 class="under-lines">Frameworks<i class="fa-solid fa-chevron-right solid"></i></h3>
+          <h3 class="under-lines">Skills<i class="fa-solid fas fa-chevron-right solid"></i></h3>
+        </div>
```

Comment:
- [ ] Please make sure to implement all the changes requested previously as you seem to have missed a couple. I've attached the [previous comment](https://github.com/Isaac-ej/Portfolio_Website/pull/8#discussion_r1218714145) below for the above code snippet regarding using a `ul` tag so please work on them 👇
> Great job! 👏 You have added all the techs, and I appreciate the way your inner list looks. However, there are a couple of improvements to consider:
> 
>     1. List Structure:
>        
>        * Since it is a list inside a list, make sure to wrap the `<li>` tag with a `<ul>` tag. This will ensure proper nesting and maintain the hierarchical structure of the list. 📜
> 
>     2. Parent List Structure:
>        
>        * Currently, you are using a `<div>` and `<h3>` for the parent list. To maintain consistency, consider using `<ul>` and `<li>` tags for the parent `<div>`. This will align with the structure of your nested lists. 📊
> 
> 
> By making these updates, you will enhance the overall structure and organization of your lists. Keep up the good work! 💪